### ISSUE TITLE
0.4.0 is "destroying" the tensorflow module import -> `AttributeError: 'NoneType' object has no attribute 'keras'`

### ISSUE THREAD

**Issue by tarrade** [lckr/jupyterlab-variableInspector#131]
on Mac and Debian. the version 0.4.0 of this jupyterlab extension is "destroying" the tensorflow module import.

You can see in the example below:
- import tensorflow in a cell  and doing some tensorflow code is working
- executing the same code in the next cell is not working anymore
`AttributeError: 'NoneType' object has no attribute 'keras'`

it this this plugin is chaning tf in NoneType object:

![image](https://user-images.githubusercontent.com/12021701/76498946-22093480-643e-11ea-96ad-ffe0121cc202.png)

I am using jupyter lab 1.2.7 and jupyter la is running using the following packages:
https://github.com/tarrade/proj_custom_ai_platform_notebook/blob/master/env/jupyter-notebook.yaml

the env to run Tensorflow 2.1.0 code is here:
https://github.com/tarrade/proj_multilingual_text_classification/blob/master/env/environment.yml

I tried many version of Jupyter Lab and I try to enable/disable every plugin. The only solution I found was to disable this jupyterlab extension.

There is no error or warning on jupyter lab logs

**Comment by derwind** [comment]
I've been faced to the same issue.
My environment is:
* OS: Debian 9 (using GCP's 'Deep Learning VM Image')
* TensorFlow 2.1.0
* JuyterLab 2.1.1
* jupyterlab-variableInspector 0.5.0

After whole day investigation, I also reached the same conclusion: a solution is disabling this extension. And I found remarkable logs in JuyterLab.

If this extension is enabled, following logs appear immediately after notebooks are opened:
```
2020-04-26 07:14:47.351116: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libnvinfer.so.6
2020-04-26 07:14:47.352801: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libnvinfer_plugin.so.6
```

On the other hand, if this extension is disabled, these logs appear when executing `import tensorflow as tf`.

I guess that `libnvinfer.so.6` and `libnvinfer_plugin.so.6` may be loaded into memory earlier than expected in case of problems.
